project_templates/fastapi_safir_app/example/src/example/models/metadata.py

```diff
@@ -0,0 +1,20 @@
+"""Metadata models for example."""
```

Comment:
As all endpoints are gathered into a single module; should all Pydantic models be gathered into a single module? And should the Metadata model be renamed to "Index" since it's the model returned by the "/" endpoints, as opposed to the more specific model for the "metadata" key, which is standardized by `safir.metadata.Metadata` ?